openpbs.spec

```diff
@@ -181,18 +187,24 @@ Requires: libical
 Autoreq: 1
 
 %description %{pbs_server}
-PBS Professional® is a fast, powerful workload manager and
+PBS Workload Manager is a fast, powerful workload manager and
 job scheduler designed to improve productivity, optimize
 utilization & efficiency, and simplify administration for
 HPC clusters, clouds and supercomputers.
 
 This package is intended for a server host. It includes all
-PBS Professional components.
+OpenPBS components.
```

Comment:
In the unlikely event the name changes again in the future, perhaps we should simply use PBS rather than OpenPBS.